jwpincus/punstartr
# Analytics Using D3

#### What does  this PR do?

Adds a new API endpoint for count of projects by category
Adds an analytics page
Adds a bar chart for number of backers for projects by consuming internal API via AJAX
Adds a pie chart for breakdown of projects by category by consuming internal API via AJAX

#### Where should the reviewer start?

Review graphs.js (it has all the D3 and AJAX calls in it)

#### How should this be manually tested?

Spin up server locally, navigate to Analytics page via navbar
Ensure graphs populate

#### What are the relevant tickets?

https://www.pivotaltracker.com/story/show/146546709

#### Screenshots

![jun-07-2017 16-30-16](https://user-images.githubusercontent.com/4068853/26910091-0abff490-4bc2-11e7-8a1a-237db4b52811.gif)

#### Questions:
  - Do Migrations Need to be ran? 
  - Do Environment Variables need to be set? 
  - Any other deploy steps? 

Nope!
